File: installation.py
import os
import shutil

home_dir = os.path.expanduser('~')

def copy_folder(source, destination):
    if not os.path.exists(destination):
        os.makedirs(destination)

    for item in os.listdir(source):
        source_path = os.path.join(source, item)
        destination_path = os.path.join(destination, item)
        
        if os.path.isdir(source_path):
            copy_folder(source_path, destination_path)
        else:
            shutil.copy2(source_path, destination_path)

source_folder = 'scripts'
destination_folder = f"{home_dir}/my_scripts"

copy_folder(source_folder, destination_folder)
print('Successfully executed.')

File: installation.js
const fs = require("fs");
const path = require("path");
const os = require("os");

const homeDirectory = os.homedir();

function copyFolderSync(source, destination) {
  if (!fs.existsSync(destination)) {
    fs.mkdirSync(destination, { recursive: true });
  }

  const entries = fs.readdirSync(source, { withFileTypes: true });

  for (let entry of entries) {
    const sourcePath = path.join(source, entry.name);
    const destinationPath = path.join(destination, entry.name);

    entry.isDirectory()
      ? copyFolderSync(sourcePath, destinationPath)
      : fs.copyFileSync(sourcePath, destinationPath);
  }
}

const sourceFolder = "scripts";
const destinationFolder = homeDirectory + "/my_scripts";

copyFolderSync(sourceFolder, destinationFolder);
console.log("Successfully executed.");
